lib/resources/r4/related_person_patch.yaml

```diff
@@ -65,6 +65,40 @@ operations:
         <li>If `line` is provided, only the first four values are used. Additional values will be ignored.</li>
       </ul>
 
+  - name: add-identifier
+    path: /identifier/-
+    operation: add
+    type: Identifier
+    url: https://hl7.org/fhir/R4/related-person-definitions.html#RelatedPerson.identifier
+    description: Add a new identifier for the related person.
+    example: |
+      {
+        "path": "/identifier/-",
+        "op": "add",
+        "value": {
+          "type": {
+            "coding": [
+              {
+                "code": "MR",
+                "system": "http://hl7.org/fhir/v2/0203"
+              }
+            ]
+          },
+          "system": "2.16.840.1.113883.6.1000",
+          "value": "&lt;unique identifier&gt;",
+          "period": {
+            "start": "2016-01-02T00:00:00-05:00",
+            "end": "2020-01-02T00:00:00-05:00"
+          }
+        }
+      }
+    note: >
+      <ul>
+        <li>Must not contain the `use` field.</li>
+        <li>Must contain the `type`, `system`, and `value` fields.</li>
+        <li>When specifying a period, the fields must include a time component with a timezone.</li>
```

Comment:
nit: you can add back in the `period` formatting you had before like you are doing for `use`, `type`, etc. above.

Reply:
good catch, updated [here](https://github.com/cerner/fhir.cerner.com/pull/555/commits/60fbd8b256ad4603508b7cc18c9da8700d58d365#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dL99)